views/index.erb

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <center>
+  <head>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <title>Feed</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  </head>
+
+  <body>
+    <div class="container">
+      <div class="alert alert-info" role="alert">
```

Comment:
you've got all this stuff duplicated in your index.erb and homepage.erb. not very DRY!

you can create a `layout.erb` file with a `<%= yield %>` command in it, and Sinatra will wrap every page in your app with it. you can use this to extract the stuff that's the same on every page into a single place. i found this lesson very helpful to understand this: <https://learn.co/lessons/sinatra-yield-readme>